docker-compose.yml

```diff
@@ -16,5 +16,11 @@ services:
       - ./:/app
     profiles: ["exclude-from-up"]
 
-  redis:
-    image: "redis/redis-stack-server:latest"
+  redisStack:
+    container_name: redisStack
+    hostname: redisStack
```

Comment:
```suggestion
```
These are not needed, docker compose should create the hostname based on the above redis-stack